_locales/_languages.json

```diff
@@ -29,6 +29,7 @@
   "pl": "Polski",
   "pt": "Português",
   "pt_BR": "Português do Brasil",
+  "ro": "română",
```

Comment:
```suggestion
  "ro": "Română",
```

Reply:
not sure how Romanian does capitalisation, but I guess you did the research

Reply:
assuming that proper nouns and language names at the beginning of a phrase are written in upper case is a good and reasonable default. exceptions can be handled then :)

but yes, i am also looking is used on comparable lists - and if questions pop up, i ask the translator who is doing the translation.